store/rootmulti/store.go

```diff
@@ -714,3 +727,13 @@ func SimpleHashFromMap(m map[string][]byte) []byte {
 
 	return mm.hash()
 }
+
+func SimpleProofOpsFromMap(m map[string][]byte) map[string]merkle.SimpleValueOp {
```

Comment:
No need to convert all the `merkle.SimpleProof`s into `merkle.SimpleValueOp` when we just use one of them.

Unless you actually use the multi-return value elsewhere (where this code would make sense), why not just:

```go
// in real code handle the missing case better
func SimpleProofOpFromMap(m map[string][]byte, key string) merkle.SimpleValueOp {
    _, proofs, _ := merkle.SimpleProofsFromMap(m)
    return merkle.NewSimpleValueOp([]byte(key), proofs[key])
}
```